Poco::Zip - Requires Poco::Net for building, but only in CMake

**Describe the bug**
When trying to build Poco::Zip testrunner with CMake, I get an error saying that Poco::Net library was not found. After that, I've checked files for other build systems (progen and Makefile) and it turned out that they do not require this library for building. Most likely, this library is required by mistake (I don't see any other reason for using the Poco::Net library to run Zip archive tests).

**To Reproduce**
Build Poco::Zip with CMake (ENABLE_ZIP and ENABLE_TESTS opts set to ON)

**Logs**
```cmake
CMake Error at Zip/testsuite/CMakeLists.txt:32 (target_link_libraries):
  Target "Zip-testrunner" links to:

    Poco::Net

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
```

**Please add relevant environment information:**
 - OS: Debian 11
 - POCO Version: Latest in main (1.14?)

--- a/Zip/cmake/PocoZipConfig.cmake
+++ b/Zip/cmake/PocoZipConfig.cmake
@@ -1,5 +1,3 @@
 include(CMakeFindDependencyMacro)
 find_dependency(PocoFoundation)
-find_dependency(PocoUtil)
-find_dependency(PocoXML)
 include("${CMAKE_CURRENT_LIST_DIR}/PocoZipTargets.cmake")

--- a/Zip/testsuite/CMakeLists.txt
+++ b/Zip/testsuite/CMakeLists.txt
@@ -29,4 +29,4 @@ else()
 		COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}/data ${CMAKE_CURRENT_BINARY_DIR}/data
 	)
 endif()
-target_link_libraries(Zip-testrunner PUBLIC Poco::Zip Poco::Net CppUnit)
+target_link_libraries(Zip-testrunner PUBLIC Poco::Zip CppUnit)

